src/briefcase/commands/new.py

```diff
@@ -420,7 +420,7 @@ def build_app_context(self):
 
         gui_framework = self.input_select(
             intro="""
-What GUI toolkit do you want to use for this project?""",
+What GUI toolkit do you want to use for this project? Note: PySide2 and PursuedPyBear cannot be deployed to Android or iOS""",
```

Comment:
Ok, I'll use that for the options. As for how it would be processed, I'm thinking of just extracting out the corresponding string. eg: 'Pyside2' from the string 'PySide2 (does not support iOS/Android deployment)'. Or should I update the names used in the briefcase template instead?